vue-app/src/store/index.ts

```diff
@@ -88,6 +90,7 @@ interface RootState {
   committedCart: CartItem[]
   contribution: BigNumber | null
   contributor: Contributor | null
+  hasVoted: boolean
```

Comment:
Would this make more sense to track this on `currentUser.hasVoted`?

Reply:
Although I suppose we already have a mix of top-level state related to the user, e.g. `contribution` 🤷‍♂️

Reply:
Hmm yes. I didn't think of that. Makes total sense. I'm going to add myself a ticket into the Tech Debt epic to refactor this.

Reply:
https://www.notion.so/efdn/Refactor-hasVoted-state-8b1fa27fa869410ba06baa248a7fb78e